src/attr.c

```diff
@@ -274,8 +277,9 @@ int store_geo_attr(struct ticket_config *tk, const char *name, char *val, int no
 	if (!notime)
 		get_time(&a->update_ts);
 
+	assert(strnlen(name, BOOTH_NAME_LEN) < BOOTH_NAME_LEN);
```

Comment:
OK, I understand that it might be too easy (accidentally or not) to push booth layer out of the function with a single crafted geo attribute in CIB/crm_ticket one-liner. So does turning that into warning sounds better? Then, we should likely warn also in case of trying to set/delete such a troublesome (too long in either name or value) attribute being provided by the user to geostore command.